oracle/datadog_checks/oracle/oracle.py

```diff
@@ -14,6 +14,7 @@
 from . import queries
 
 EVENT_TYPE = SOURCE_TYPE_NAME = 'oracle'
+MAX_CUSTOM_RESULTS = 100
```

Comment:
Is there a reason? If anything, let's make this configurable

Reply:
Same, from the postgres integration. I doubt you'd want more than 100 rows for a single metrics?

Reply:
RIght I copied the wrong thing. Does it make sense as a safe guard?